atomic_weights.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+import os
+from carsus.io.nist.weightscomp import download_weightscomp
```

Comment:
Small thing- can you run isort on both of these files?

Reply:
Ohh! I was wrong. It's working. I have modify the code